apps/src/music/views/MusicView.jsx

```diff
@@ -109,12 +109,15 @@ class UnconnectedMusicView extends React.Component {
       Globals.setPlayer(this.player);
     });
 
-    this.loadInstructions().then(instructions => {
-      this.setState({
-        instructions: instructions,
-        showInstructions: !!instructions
+    // Disable instructions for now.
```

Comment:
Hm should this be behind a param in `appConfig` so we can still turn it back on if needed?

Reply:
Good call; will do.